### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by matteoettam09**

### CODE REVIEW

File: requirements.txt
Comment (xaviergilbert) [inline requirements.txt:1]: I observed a break in a strategy with a pyarrow version.
I think a good practice would be to not jump over major versions
pyarrow >=10.0.1, <12.0.1 -> pyarrow >=10.0.1, <11.0.0

Comment (xaviergilbert) [inline requirements.txt:1]: I observed a break in a strategy with a pyarrow version.
I think a good practice would be to not jump over major versions
pyarrow >=10.0.1, <12.0.1 -> pyarrow >=10.0.1, <11.0.0

Comment (Caceresenzo) [reply requirements.txt:1]: Pyarrow is an exception.
It don't have a lot of minor/patch version, and goes quickly on majors.

See: https://pypi.org/project/pyarrow/#history

**Review by Caceresenzo — APPROVED** [review]
Be careful next time.
It pains me to see your first 3 commits.

**Review by xaviergilbert — APPROVED** [review]

**Pull request merged**